tests/test_parser.py

```diff
@@ -2482,6 +2482,51 @@ def test_Xiaomi_MS1BB_MI_obj4a13():
         },
     )
 
+def test_Xiaomi_RS1BB_MI_obj4806():
+    """Test Xiaomi parser for Linptech RS1BB(MI) with obj4806."""
+    data_string = b"\x04>)\x02\x01\x00\x00gL\xb98\xc1\xa4\x1d\x02\x01\x06\x19\x16\x95\xfeXY\x0f?JgL\xb98\xc1\xa4\xd6\xe5{\x83\x04\x00\x00\xd0\x1e\x0bK\xc0"
```

Comment:
The `data_string` has to only contain the actual service load, without UUID and without RSSI (last byte). 
The service load starts with `x19\x16\x95\xfe` (19 = length, 16 = service load, 95 FE = UUID of xiaomi)

So, most I think you should use 
`data_string = b"XY\x0f?JgL\xb98\xc1\xa4\xd6\xe5{\x83\x04\x00\x00\xd0\x1e\x0bK\xc0"`

Reply:
Thanks for the hint! I was wondering how come the test data was far longer in this case compared to the other devices :)